pages/docs/integrations/grouparoo-intercom.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: "@grouparoo / intercom"
-date: "2021-10-12"
+title: "Intercom"
```

Comment:
👍 we can make the title shorter since we list the npm package name `@grouparoo/intercom` later on the page